oaknational/Oak-Web-Application
# fix(how-to): correct the heading hierarchy on the develop curriculum …

…page

## Description

- change heading to h5 tag

## Issue(s)

Fixes #1363 

## How to test

1. Go to develop your curriculum
2. Click on dev tools
3. You should see hows tos are h5 tags

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
